array_methods_lab.js

```diff
@@ -0,0 +1,308 @@
+// # Exercises
+//
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+//
+// ```
+// Input: [1,2,3,4,5]
+// Output: [11,12,13,14,15]
+// ```
+// USE MAP METHOD
+let arr1 = [2, 3, 4, 11, 5]
+let increasedBy10 = arr1.map((currentElem)=>{
+  return currentElem + 10
+})
+console.log(increasedBy10)
+
+// 2. Given an array, return a new array where every non-string element is converted to an empty string.
+//
+// ```
+// Input: ["a", 123, "b", "c", {name: "cat"}]
+// Output: ["a", "", "b", "c", ""]
+// ```
+// // USE MAP METHOD
+let mixedArr = [1, 'a', 4, 'oops', {1: 'objectValue'}]
+let num2 = mixedArr.map((currentElem) => {
+  // console.log(typeof(currentElem))
+  if(typeof(currentElem) === 'string'){
+    return currentElem
+  }else{
+    return ''
+  }
+
+})
+console.log(num2)
```

Comment:
So you have the right idea! However, It looks like your function is actually doing the opposite. How can you negate you conditional?